ThePorch/src/AppStateTracker.js

```diff
@@ -0,0 +1,51 @@
+import { Component } from 'react';
+import { AppState } from 'react-native';
+import PropTypes from 'prop-types';
+
+class AppStateTracker extends Component {
+  state = {
+    appState: AppState.currentState,
+    currentTime: new Date().getTime(),
+  };
+
+  componentDidMount() {
+    this.props.track({ eventName: 'App Opened' });
+    AppState.addEventListener('change', this.handleAppStateChange);
```

Comment:
Also....does Amplitude not track this out of the box

Reply:
Not that I can tell. If it's tracked, I can't find it in the dashboard / when building a chart.